library/core/src/intrinsics.rs

```diff
@@ -1732,6 +1732,11 @@ extern "rust-intrinsic" {
     /// See documentation of `<*const T>::guaranteed_ne` for details.
     #[rustc_const_unstable(feature = "const_raw_ptr_comparison", issue = "53020")]
     pub fn ptr_guaranteed_ne<T>(ptr: *const T, other: *const T) -> bool;
+
+    /// Allocate at compile time. Should not be called at runtime.
+    #[rustc_const_unstable(feature = "const_heap", issue = "none")]
```

Comment:
I guess we need a tracking issue for the experimental implementation of https://github.com/rust-lang/const-eval/issues/20 now